HW0/P4/P4.py

```diff
@@ -68,9 +97,50 @@
 
     # Initial conditions for s0 and Sigma0
     # Compute the rest of sk using Eqs (2), (3), (4), and (5)
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '-r', label='Filtered trajectory')
+    C = np.array([[rx,0,0,0,0,0],[0,ry,0,0,0,0],[0,0,rz,0,0,0]])
```

Comment:
Better ways to construct matrix B and C by calling some functions provided by numpy:
B = np.diag((bx, by, bz, bvx, bvy, bvz))
C = np.hstack([np.diag((rx,ry, rz)), np.zeros((3,3))])

This looks more elegent and more scalabe than simply hardcoding :)